crates/wasm-bridge-js/src/conversions/into_closure.rs

```diff
@@ -7,7 +7,7 @@ use crate::*;
 pub(crate) type MakeClosure<T> = Box<dyn Fn(DataHandle<T>) -> (JsValue, DropHandler)>;
 
 pub trait IntoMakeClosure<T, Params, Results> {
-    fn into_make_closure(self) -> MakeClosure<T>;
+    fn into_make_closure(self, engine: Engine) -> MakeClosure<T>;
```

Comment:
It is probably more "correct" to "propagate" the same engine into the `Caller` like this, but the `Engine` struct is completely unused and is only there for compatibility with wasmtime, so I think it would just needlessly bloat the code.

Reply:
Engine is used to construct the store in caller, which leads to impl AsContext and AsContextMut for Caller